modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -40,7 +40,12 @@ where:
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
 -e DISCOVERY_MODE=true -e FEATURES=performance -e IMAGE_REGISTRY="<disconnected_registry>" \
 -e CNF_TESTS_IMAGE="cnf-tests-rhel8:v{product-version}" \
+<<<<<<< HEAD
 /usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+=======
+-e LATENCY_TEST_RUNTIME=<time_in_seconds> \
+<disconnected_registry>/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh --ginkgo.v --ginkgo.timeout="24h"
+>>>>>>> 8875e10430 (OCPBUGS-25695: Ginkgo test suite now times out in 1hr. Setting flag to 24hr to accomodate latency tests)
```

Comment:
Please resolve conflict here.

Reply:
Whoops, thanks for catching, thought I resolved them all.